### ISSUE TITLE
Cannot connect with secure websocket

### ISSUE THREAD

**Issue by luisfmcalado** [processone/tsung#309]
The secure websocket isn't working when the server type is `websocket_ssl`. 

```xml
<?xml version="1.0"?>
<!DOCTYPE tsung SYSTEM "/usr/local/share/tsung/tsung-1.0.dtd">
<tsung loglevel="debug" dumptraffic="true" version="1.0">

  <clients>
    <client host="127.0.0.1" maxusers="4" use_controller_vm="true"/>
  </clients>

  <servers>
    <server host="lvh.me" port="443" type="websocket_ssl" weight="1" />
  </servers>

  <load>
    <arrivalphase phase="1" duration="10" unit="second">
      <users maxnumber="1" arrivalrate="10" unit="second" />
    </arrivalphase>
  </load>
 
  <options>
    <option name="websocket_path" value="/mqtt"/>
    <option name="websocket_subprotocols" value="mqtt"/>
  </options>

  <sessions>
    <session name="mqtt-subscribe" weight="1" type="ts_mqtt">
      <request>
        <mqtt type="connect" username="usernameX" password="" clean_start="true" timeout="30" keepalive="3600"></mqtt>
      </request>

      <request>
        <mqtt type="subscribe" topic="/#" qos="1"></mqtt>
      </request>

      <for from="1" to="8000" incr="1" var="counter">
        <request>
          <mqtt type="waitForMessages" timeout="10800"></mqtt>
        </request>
      </for>

      <request>
        <mqtt type="disconnect"></mqtt>
      </request>
    </session>
  </sessions>
</tsung>
```

### Expected 
The users must connect to MQTT and subscribe to topic `/#` using ssl termination in the load balancer.

### Result
The users cannot connect to MQTT and the result is `Error: Unable to send data, reason: timeout`

**Comment by luisfmcalado** [comment]
@nniclausse Can you please check this issue. I have open a PR that solves the problem, can you please take a look at it. 
Thank you.

**Comment by tisba** [comment]
Fixed by #310, thanks @luisfmcalado!
